extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -146,7 +149,7 @@ public function createNewBoardModal() {
 			'method' => 'post',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "method" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)